requirements/bench.txt

```diff
@@ -14,3 +14,4 @@ torchvision
 pyomo>=5.7
 IOHexperimenter>=0.2.8
 mixsimulator>=0.2.9.5
+tensorflow
```

Comment:
I'm a bit uneasy at adding yet another (huge) requirement :s Let's add it for now but I may revert to an inline import eventually. In the meantime please add a version.

Reply:
Agree. I have removed it from the requirements, however, I don't want to crash the ci, so I have to correct that.

Reply:
For the CI you might simply, if the inline import does not work, skip the benchmark.